### DISCUSSION TITLE
GSoC 2024

### DISCUSSION THREAD

**Discussion by Preetam-Das26** [fury-gl/fury#859] [GSoC/GSoD]
Hello,
I am Preetam from India.I am a Btech undergrad at NIT Rourkela interested in Python,Django,HTML,C, C++,CSS, JS,JAVA, git,Github,etc. I am interested in this field and have worked on some few projects in these areas involving ML/AI using python libraries like tensorflow,numpy,pandas,scikit-learn,keras and python backend with django.I look forward to my participation in GSoC 2024 in your organisation.If possible please let me know if how to contribute as a beginner to make this journey smooth. Also to get selected as GSoC project contributor which steps should be taken and how to begin?
Thank you

**Comment by robinroy03** [comment]
Hi Preetam, this [blog](https://levelup.gitconnected.com/my-journey-guide-for-google-summer-of-code-7b03e1007c07) will help you get started.

Also, join FURY [discord](https://discord.gg/GhxPcRfH).
